node/src/reactor/validator.rs

```diff
@@ -761,6 +841,10 @@ impl reactor::Reactor for Reactor {
                             warn!("received get request for gossiped-address from {}", sender);
                             return Effects::new();
                         }
+                        Tag::BlockHeaderByHash => todo!("Handle GET BlockHeaderByHash response"),
+                        Tag::BlockHeaderAndFinalitySignaturesByHeight => {
+                            todo!("Handle GET BlockHeaderByHeight response")
+                        }
```

Comment:
We need to avoid `todo!`s here.  A malicious peer could trigger this easily.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/1033/commits/c8a1df3e5747eb7b2ffda10af05500651ab8a206